assets/js/fitvids.js

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-inline-comments */
+const fitVids = (postContent, options) => {
```

Comment:
```suggestion
if (window) {
 window.fitVids = (postContent) => {
```
I noticed that Babel put a few other functions in the global scope.  Probably fine, but I figured this was a little safer.

Also, since we're not using `options` can we just remove it entirely?